packages/@lwc/errors/src/compiler/error-info/template-transform.ts

```diff
@@ -839,4 +839,11 @@ export const ParserDiagnostics = {
         level: DiagnosticLevel.Error,
         url: '',
     },
+
+    SINGLE_IF_DIRECTIVE_ALLOWED: {
+        code: 1182,
+        message: 'Only one If directive is allowed. The rest are ignored.',
```

Comment:
Note: the actual code is passing the if: directive into the error message but it isn't being used here (with `{0}`).

Also, suggested improvement: passing the tagname into the error message.
```
Multiple if: directives found on 'c-custom-element'. Only one if: directive is allowed; the rest are ignored.
```